can/interfaces/nixnet.py

```diff
@@ -120,16 +118,18 @@ def __init__(
             self.__session_send.start()
             self.__session_receive.start()
 
-        except errors.XnetError as err:
-            raise NiXNETError(function="__init__", error_message=err.args[0]) from None
+        except errors.XnetError as error:
+            raise CanInitializationError(
+                f"{error.args[0]} ({error.error_type})", error.error_code
+            ) from None
```

Comment:
`raise from None` is to explicitly remove the earlier exception from the traceback right?

Reply:
Yes. 😄 But all the information should now be in the new exception. That's also how it was before.